src/modbus-rtu.c

```diff
@@ -1112,6 +1114,38 @@ int modbus_rtu_set_rts_delay(modbus_t *ctx, int us)
     }
 }
 
+int modbus_rtu_get_recv_filter(modbus_t *ctx)
+{
+    if (ctx == NULL) {
+        errno = EINVAL;
+        return -1;
+    }
+
+    if (ctx->backend->backend_type == _MODBUS_BACKEND_TYPE_RTU) {
```

Comment:
Is this check meaningful? can you actually reach this code path without having this as the backend?  (same for the setter)